activerecord/lib/active_record/sanitization.rb

```diff
@@ -143,8 +143,9 @@ def disallow_raw_sql!(args, permit: connection.column_name_matcher) # :nodoc:
 
         if unexpected
           raise(ActiveRecord::UnknownAttributeReference,
-            "Query method called with non-attribute argument(s): " +
-            unexpected.map(&:inspect).join(", ")
+            "Query method called with non-attribute argument(s): " \
+            "#{unexpected.map(&:inspect).join(", ")}" \
```

Comment:
Shouldn’t this include an extra space to be seperated from the next sentence?

```suggestion
            "#{unexpected.map(&:inspect).join(", ")}. " \
```

Reply:
Good.

Reply:
@sunny I think it was done in 19489f5